theia-rust-docker/README.md

```diff
@@ -0,0 +1,41 @@
+# Theia with Rust support
+
+## Building the image
+
+RUST\_CHANNEL=nightly make image
```

Comment:
Would be good to add a bit of context: it's assumed that one is in the `theia-rust-docker` folder when running this command, I think? 

e.g.: 
```
cd <this repo>/theia-rust-docker
RUST_CHANNEL=nightly make image
```

Reply:
Updated it.